websites/W/wumpus.store/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "998532462755979275",
+	}),
+	browsingUnix = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/KaW8pr2.png",
+			details: "Viewing Home Page",
+			startTimestamp: browsingUnix,
+		},
+		{ pathname, origin } = document.location;
+
+	let title: HTMLElement, logo: HTMLElement;
+
+	switch (pathname) {
+		case "/bot/add": {
+			presenceData.details = "Adding a new bot";
+			break;
+		}
+		case "/search": {
+			presenceData.details = "Searching for a new bot";
```

Comment:
Optional but you could add a small image key, all images can be found [here](https://github.com/PreMiD/Presences/blob/b861c896ac477bd51e91614138152f1db20a9579/%40types/premid/index.d.ts#L78)
```suggestion
			presenceData.details = "Searching for a new bot";
			presenceData.smallImageKey = Assets.Search
```

Reply:
Ohh great idea I will add that tomorrow!